test_cases/List_of_U.S._stadiums_by_capacity.json

```diff
@@ -0,0 +1,3310 @@
+{
+  "name": "wikipedia scrape: List_of_U.S._stadiums_by_capacity",
+  "description": "",
+  "priorityThresh": "",
+  "source": "Wikipedia",
+  "normalizers": {
+    "name": [
+      "toLowerCase",
+      "removeOrdinals",
+      "stripPunctuation",
+      "abbreviateDirectionals"
+    ]
+  },
+  "tests": [
+    {
+      "id": "Kyle Field",
+      "status": "pass",
+      "type": "wiki-poi",
+      "in": {
+        "text": "Kyle Field"
+      },
+      "expected": {
+        "coordinates": [
+          -96.34028,
+          30.61
+        ],
+        "distanceThresh": 5000,
+        "properties": {
+          "layer": "venue"
+        }
+      }
+    },
```

Comment:
We will probably want an expected `name` field. Right now all these tests will pass as long as any venue within 5km of the given point is returned. While that's nice for getting around minor name differences it's probably too lenient :(